File: C/URI/1065.c
#include <stdio.h>
 
int main() {
 
  int n, numero, par;
        n = 5;
        par = 0;
        while(n > 0)
        {
                scanf("%d", &numero);
                if(numero%2 == 0)
                        par = par + 1;
                else
                        ;
        n = n - 1;
        }
        printf("%d valores pares\n", par);

 
    return 0;
}

File: JS/URI/1065.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var numero;
var par =0;
for(var i=0; i<5; i++)
{
    numero = parseInt(lines.shift());
    if(numero%2==0)
    {
        par+=1;
    }

}

console.log(par+" valores pares");
